client/packages/invoices/src/Prescriptions/DetailView/SidePanel/PricingSection.tsx

```diff
@@ -16,52 +17,54 @@ export const PricingSectionComponent = () => {
   const c = useFormatCurrency();
 
   const {
-    query: { data: prescriptionData}
+    query: { data: prescriptionData },
   } = usePrescription();
   const pricing = prescriptionData?.pricing;
   if (!pricing) return null;
 
   const insuranceId = prescriptionData.nameInsuranceJoinId;
-  const { storeId } = usePrescriptionGraphQL()
+  const { storeId } = usePrescriptionGraphQL();
   const {
-    query: {data: insuranceData}
-   } = useInsurance(insuranceId, storeId);
+    query: { data: insuranceData },
+  } = useInsurance(insuranceId, storeId);
 
   return (
     <DetailPanelSection title={t('heading.pricing')}>
       <Grid container gap={0.5}>
-        
         {insuranceData && (
-        <>
-        <PanelRow>
-          <PanelLabel>{t('label.insurance-provider-name')}</PanelLabel>
-          <PanelField>
-            {insuranceData.insuranceProviders?.providerName}
-          </PanelField>
-        </PanelRow>
+          <>
+            <PanelRow>
+              <InfoTooltipIcon
+                title={t('messages.insurance-description')}
+              />
+              <PanelLabel fontWeight="bold">
+                {t('heading.insurance')}
+              </PanelLabel>
+            </PanelRow>
 
-        <PanelRow>
-          <PanelLabel>{t('label.insurance-policy-number')}</PanelLabel>
-          <PanelField>
-            {insuranceData.policyNumber}
-          </PanelField>
-        </PanelRow>
+            <PanelRow sx={{ marginLeft: '10px' }}>
```

Comment:
nit: please try and avoid using pixels for layout and fonts. We need to move to a responsive UI and fixing an absolute value for pixels prevents scaling.

For this, use `marginLeft: 1.25` which will give the same margin under default settings.